easybuild/easyconfigs/p/PyQtGraph/PyQtGraph-0.10.0-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'PyQtGraph'
+version = '0.10.0'
+
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.pyqtgraph.org/'
+description = """PyQtGraph is a pure-python graphics and GUI library built on PyQt4/PySide and numpy."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.pyqtgraph.org/downloads/%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['c74597dd87c31987099bd0949a5207e5d58297bd8a663279562ca1cb334074fc']
+
+dependencies = [
+    ('Python', '3.6.4'),
+    ('PyQt5', '5.9.2', versionsuffix),
+]
+
```

Comment:
Please add `use_pip = True`

Reply:
@vanzod, I'll do this in a new commit now...